### ISSUE TITLE
RTE toolbar disappears when pasting text

### ISSUE THREAD

**Issue by ordo123** [umbraco/Umbraco-CMS#14193]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

11.3.0

### Bug summary

When pasting in text or writing inside a RTE block create in the grid editor (legacy), the toolbar disappears.

The toolbar only disappears when you have enough text so it is out of view.

The only way to get the toolbar back (which I have found) is to resize or move the browser window.

### Specifics

https://user-images.githubusercontent.com/73296766/235676532-42409b43-531e-4ea5-bc38-6ef5e10bd59c.mp4

The first block called RTE is just a normal RTE block added to the document type.
The block under with the blue bar stated as "Headline" is the RTE inside the Grid editor.

### Steps to reproduce

A clean umbraco 11 install.
(Set "ShowDeprecatedPropertyEditors": true" in appsettings.json)
Create a document type with the grid editor (legacy)
Create a new page and add a RTE block as content
Spam paste some text until the toolbar is out of view, then it disappears when further pasting text.

This problem can also occur by just writing or pressing enter when the toolbar is out of view

Before pasting text:
![image](https://user-images.githubusercontent.com/73296766/235671818-eef8ba9d-e2c9-45be-8753-d70519f2c1ac.png)

After pasting text when the toolbar is out of view (eg when scrolled down as the toolbar always stays on top of the RTE block).
![image](https://user-images.githubusercontent.com/73296766/235672232-7265c9d3-2832-4011-b952-ad953e2b0f52.png)

### Expected result / actual result

I expect the toolbar to not disappear.
I found this problem by migrating a site from v8 to v11. But it is fully reproducible on a clean install. I am not in a position where I can migrate to the block editor.

**Comment by andr317c** [comment]
Hey, thanks for the great reproduction steps. I can certainly reproduce the issue.

In V10 it works, however, the screen does not follow the text while it's being copied and going out of the screen. Here is a GIF:
![V10IssueRTEToolbarDoesNotDissapear](https://user-images.githubusercontent.com/73799582/236783180-11d79864-806f-4f76-bd4c-2e2079461371.gif)


In V11 it does not work. Here is a GIF of me reproducing the issue: 
![V11IssueRTEToolbarDissapears](https://user-images.githubusercontent.com/73799582/236784844-de8a73fa-a88c-4867-808a-50d772ccfe94.gif)


I will mark this issue as up for grabs because we have good reproduction steps and it works in V10. So we hopefully can take some inspiration from that.

**Comment by ordo123** [comment]
Thank you! It should also be noted that i v10 the toolbar follows when scrolling down. In v11 the toolbar does not follow which makes it very annoying to edit text further down.

**Comment by iOvergaard** [comment]
For the sake of posterity, I just want to put here, that v10 uses TinyMCE 4 and v11 saw an upgrade to TinyMCE 6, which comes with a different API. Some things were sacrificed on our end to try and coerce the editor into working as it used to in v10. I think we could do a lot of good work by embracing the default behavior of the new TinyMCE editor and try and remove some of our "hacks".

As for the Grid Layout RTE, there are certainly some hacks in place to make TinyMCE work in a more inline fashion. Any improvements made to the general editor should probably be set to flow down to the Grid Layout editor, which is being bootstrapped through its own AngularJS view template, and hopefully that will make it work better.

Please note that the Grid Layout editor is deprecated and will be removed in a future version of Umbraco yet to be determined.

**Comment by ordo123** [comment]
@iOvergaard 

> Any improvements made to the general editor should probably be set to flow down to the Grid Layout editor, which is being bootstrapped through its own AngularJS view template, and hopefully that will make it work better.

Is this something that is going to be "fixed" in an upcomming update? Unfortunately I lack the knowledge of creating a PR for this myself.

**Comment by alandraper** [comment]
Yeah, our website editor is very frustrated at the disappearing toolbar.

**Comment by jattwood** [comment]
Experiencing the same issue in V12 using the grid editor. After you insert content into a grid editor using RTE, the toolbar disappears and never re-appears. This makes using grid editor rte un-usable. Any suggestions on where we can look to fix this (other than removing the grid editor)?

**Comment by jaandrews** [comment]
I've run into this as well in my umbraco 11.4.2 installation. The problem appears to be that there "tox-editor-dock-fadeout" class that is not removed for some reason on the toolbar.
![rte-bug](https://github.com/umbraco/Umbraco-CMS/assets/5932368/0c8a973a-605b-4854-9888-0fd19eab036b)
If I remove those styles with the developer tools, the toolbar works as expected, so there appears to be a problem with how the rte animates in and out.

Not sure what needs to change in the rte control to fix this, but a quick hack to address the issue would be to add styles to the backoffice to overwrite this class. Something like
``` css
.tox-editor-dock-fadeout {
    visibility: visible !important;
    opacity: 1 !important;
}
```
This might cause problems elsewhere, but it would make toolbar stay visible in the grid.

**Comment by jaandrews** [comment]
@jattwood I thought about trying the "inline" setting, but I haven't been able to make in the past for the sites I build. I have custom functionality that works similarly to the retired [Doc Type Grid Editor](https://our.umbraco.com/packages/backoffice-extensions/doc-type-grid-editor/) plugin and that mode doesn't work when there are multiple grid controls on the page (needed for accordion and tabs content). Certainly an option for most builds though.

**Comment by iOvergaard** [comment]
This should be fixed with #15595 which we expect to release with Umbraco 13.2. Thanks for reporting this! #h5yr
